addons/mail/static/src/components/message_reply/message_reply.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.MessageReply" owl="1">
+        <small class="o_MessageReply text-small pt-1 pr-2 position-relative" t-on-click.stop="highlightReply">
```

Comment:
Also when you dispatch to model, should not immediately deduce the business logic here.
So instead of dispatching to model as "highlight reply", prefer "on click reply". The spec/feature starts with "on click reply", not "highlight reply". As a reminder, components are dumb and just reflect UI from model, they don't take decisions for how user interactions should change the state of the application as a whole.

Reply:
Also we try as much as possible not to stop propagation of event. This is a selfish approach that makes code maintenance harder than it should. For example, while debugging, it's not fun to find the culprit that stop the propagation of event when we would have like the event to bubble.

The better approach is to mark event as handled, and let everyone else decide whether it should manage the event or just ignore (and let bubble). In discuss code, we do this with utils function `isEventHandled` & `markEventHandled`.